rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/are_all_terminal_types_CAV_with_none_equal_to_null.py

```diff
@@ -0,0 +1,32 @@
+from rct229.data.schema_enums import schema_enums
```

Comment:
I'm confused. It seems to me that `are_all_terminal_types_CAV()` and `are_all_terminal_types_CAV_with_none_equal_to_null()` are logically identical.  @weilixu what am I missing here?

Reply:
`are_all_terminal_types_CAV` function allows None or `terminal_b["type"] == TERMINAL_TYPE.CONSTANT_AIR_VOLUME` to be True

`are_all_terminal_types_CAV_with_none_equal_to_null` is a more strict version of `are_all_terminal_types_CAV` that only `terminal_b["type"] == TERMINAL_TYPE.CONSTANT_AIR_VOLUME` is true.